site/src/sphinx/advanced-spring-webflux-integration.rst

```diff
@@ -0,0 +1,80 @@
+.. _`Spring Boot Reference Guide`: https://docs.spring.io/spring-boot/docs/current/reference/html/
+.. _`armeria-examples`: https://github.com/line/armeria-examples
+
+.. _advanced-spring-webflux-integration:
+
+Using Armeria as a web server of a reactive Spring application
+==============================================================
+
+A user can use Armeria as a web server of his or her reactive Spring application by adding
+`spring-boot-webflux-starter` to the dependencies.
```

Comment:
How about adding why this is useful to users? e.g. HTTP/2, leveraging all the nice bells and whistles in Armeria, etc.